torchvision/models/detection/fcos.py

```diff
@@ -501,7 +501,11 @@ def postprocess_detections(
                 topk_idxs = torch.where(keep_idxs)[0]
 
                 # keep only topk scoring predictions
-                num_topk = det_utils._topk_min(topk_idxs, self.topk_candidates, 0)
+                num_topk = (
+                    det_utils._topk_min(topk_idxs, self.topk_candidates, 0)
```

Comment:
Nit: might be worth moving this if statement inside the _topk_min method to avoid having to patch all methods again. Thoughts?

Reply:
Good point.